packages/cli/src/commands/setup/deploy/__tests__/netlify.test.js

```diff
@@ -64,8 +73,21 @@ beforeEach(() => {
 
 describe('netlify', () => {
   it('should call the handler without error', async () => {
-    const netlify = require('../providers/netlify')
-    expect(async () => await netlify.handler({ force: true })).not.toThrow()
+    const netlify = await import('../providers/netlify')
+
+    let error = undefined
+    try {
+      await netlify.handler({ force: true })
+    } catch (err) {
+      error = err
+    }
+    expect(error).toBeUndefined()
```

Comment:
Is there no `expect(...).not.toThrow()` anymore?

Reply:
I tried to keep it the same but I had to update the snapshotting logic. We're no longer allowed to call `.matchSnapshot` outside of the test function - we were doing it in the mocked `writeFilesTask` function.

If I kept it as `expect(async () => await netlify.handler({ force: true })).not.toThrow()` then the new check for if the file exists and the content matches the snapshot would fail. If I do it in the more verbose way here then everything is run and the updated snapshots work as expected.